recipes/google-cloud-cpp-core/meta.yaml

```diff
@@ -0,0 +1,364 @@
+{% set version = "2.17.0" %}
+
+{%- macro test_subpackage(feature) %}
+commands:
+  - test -f $PREFIX/lib/libgoogle_cloud_cpp_{{ feature }}.{{ version }}.dylib  # [osx]
+  - test -f $PREFIX/lib/libgoogle_cloud_cpp_{{ feature }}.so.{{ version }}  # [linux]
+  - test ! -f $PREFIX/lib/pkgconfig/google_cloud_cpp_{{ feature }}.pc  # [not win]
+  - test ! -f $PREFIX/include/google/cloud/version.h  # [not win]
+  - test ! -f $PREFIX/lib/cmake/google_cloud_cpp_{{ feature }}/google_cloud_cpp_{{ feature }}-config.cmake  # [not win]
+  - if exist %LIBRARY_PREFIX%\\include\\google\\cloud\\version.h exit 1  # [win]
+  - if exist %LIBRARY_PREFIX%\\lib\\cmake\\google_cloud_cpp_{{ feature }}\\google_cloud_cpp_{{ feature }}-config.cmake exit 1  # [win]
+{%- endmacro %}
+
+{%- macro test_devel_subpackage(feature) %}
+commands:
+  - ./run_test_feature.sh   # [not win]
+  - ./run_test_feature.bat  # [win]
+requires:
+  - {{ compiler('cxx') }}
+  - cmake
+  - ninja
+files:
+  - run_test_feature.sh
+  - run_test_feature.bat
+source_files:
+  - google/cloud/{{ feature }}/quickstart/*.cc
+  - google/cloud/{{ feature }}/quickstart/CMakeLists.txt
+{%- endmacro %}
+
+{%- macro standard_subpackage(feature) %}
+script: build-libgoogle-cloud.sh   # [not win]
+script: build-libgoogle-cloud.bat  # [win]
```

Comment:
I'd prefer calling these `install-` and not `build-`; the only build step is in `build.sh` / `bld.bat`, the rest is installation (AFAICT)

Reply:
Done.